lib/hanami/logger.rb

```diff
@@ -174,10 +172,18 @@ def _message_hash(message) # rubocop:disable Metrics/MethodLength
         end
       end
 
-      # @since 0.8.0
+      # @since x.x.x
```

Comment:
@AlfonsoUceda I'm gonna cleanup this. But remember, if a method was already existing and it only changes the implementation, we shouldn't retag with `@since x.x.x`, but keeping it the old version. 😉